docs/index.rst

```diff
@@ -9,55 +9,100 @@ If you want to build a completely different site with all the same Django
 optimizations for security, scalability, L10n, and ease of use, check out
 Mozilla's `Playdoh starter kit <http://playdoh.readthedocs.org/>`_.
 
-
 .. _install-with-docker:
 
 Quickstart
 ----------
+Want the easiest way to start contributing to AMO? Try our docker-based
+development environment.
+
+First you'll need to install docker_, please check the information for
+the installation steps specific to your operating system.
+
+.. note::
+  Docker recommends installing docker-toolbox_ if you're on OSX or
+  windows and that will provide you with the ``docker-machine`` and
+  ``docker-compose`` (mac-only).
+
+Creating the docker vm (mac/windows)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you go the docker-machine route your first step is to create a vm::
+
+  docker-machine create --driver=virtualbox addons-dev
+
+Then you can export the variables so that docker-compose  can talk to
+the docker service on the vm by running::
+
+This command will tell you what you need to do that::
```

Comment:
Cross `that`? Or is it just me, not understanding the structure of that sentence?